ibm_was/datadog_checks/ibm_was/ibm_was.py

```diff
@@ -0,0 +1,127 @@
+# (C) Datadog, Inc. 2019
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import requests
+from lxml import etree
+
+from six import iteritems
+
+from datadog_checks.base import AgentCheck, ensure_unicode, is_affirmative
+
+from . import metrics, validation
+
+
+class IbmWasCheck(AgentCheck):
+
+    SERVICE_CHECK_CONNECT = "ibm_was.can_connect"
+    METRIC_PREFIX = 'ibm_was'
+
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        AgentCheck.__init__(self, name, init_config, agentConfig, instances)
+        self.metric_type_mapping = {
+            'AverageStatistic': self.gauge,
+            'BoundedRangeStatistic': self.gauge,
+            'CountStatistic': self.monotonic_count,
+            'DoubleStatistic': self.rate,
+            'RangeStatistic': self.gauge,
+            'TimeStatistic': self.gauge
+        }
+
+    def check(self, instance):
+        validation.validate_config(instance)
+        collect_stats = self.setup_configured_stats(instance)
+        url = instance.get('servlet_url')
+
+        nested_tags, metric_categories = self.append_custom_queries(instance, collect_stats)
+        custom_tags = instance.get('tags', [])
+
+        service_check_tags = list(custom_tags)
+        service_check_tags.append('url:{}'.format(url))
+
+        data = self.make_request(instance, url, service_check_tags)
+        server_data_xml = etree.fromstring(data)
```

Comment:
can this error? is it a good idea to wrap it in a try/except to ensure that it doesn't error when it's not an xml object (for example when the url is misconfigured or something)

Reply:
Good call, will do